tests/websockets/test_graphql_transport_ws.py

```diff
@@ -962,3 +911,20 @@ async def test_subscription_errors_continue(ws: WebSocketClient):
         response = await ws.receive_json()
         assert response["type"] == CompleteMessage.type
         assert response["id"] == "sub1"
+
+
+@patch.object(MyExtension, MyExtension.get_results.__name__, return_value={})
+async def test_no_extenions(mock: Mock, ws: WebSocketClient):
```

Comment:
Small typo in the tests name. IMHO it would hurt if this test and the corresponding test below for `graphql_ws` would have a slightly more descriptive name making it more obvious what's being tested.

```suggestion
async def test_no_extensions(mock: Mock, ws: WebSocketClient):
```